src/baseline.rs

```diff
@@ -201,76 +389,6 @@ impl RegistryBaseline {
     }
 }
 
-/// To get the rustdoc of the baseline, we first create a placeholder project somewhere
-/// with the baseline as a dependency, and run `cargo rustdoc` on it.
-fn create_rustdoc_manifest_for_crate_version(
```

Comment:
Are we just moving this fn around? If so, we could reduce the diff of this PR and make review easier.

Reply:
Oh, what do you mean? Is there a way to tell GitHub's diff that some code has moved? Or do you propose to not move this function?

Reply:
I would not move it unless strictly necessary if the PR also contains functional changes.

Reply:
I see. Since this PR is pretty much reviewed, I think I prefer to leave it as it is currently and in my next PRs I'll try to do it like you suggest (splitting the PRs into functional changes and just moving code around).